playbooks/06-deploy-architecture.yml

```diff
@@ -87,14 +87,48 @@
           block:
             - name: Generate needed facts out of local files
               vars:
+                _ctl_ifaces_vars: >-
+                  {{
+                    _ctl_data | dict2items | selectattr('key', 'in', _ifaces_vars)
+                  }}
                 _ipv4_network_data: >-
                   {{
-                    _ctl_data | dict2items |
-                    selectattr('key', 'in', _ifaces_vars) |
+                    _ctl_ifaces_vars |
                     selectattr('value.ipv4.address', 'defined') |
                     selectattr('value.ipv4.address', 'equalto', _controller_host) |
                     map(attribute='value.ipv4') | first | default({})
                   }}
+                _ipv6_network_data: >-
+                  {{
+                    _ctl_ifaces_vars |
+                    selectattr('value.ipv6.address', 'defined') |
+                    selectattr('value.ipv6.address', 'equalto', _controller_host) |
+                    map(attribute='value.ipv6') | first | default({})
+                  }}
+                _ipv4_sshd_ranges: >-
```

Comment:
i do not see this `_ipv4_sshd_ranges` used by this change,

below on line 145 there is `_ipv4_network_data + _ipv6_sshd_ranges`
should there be ipv4_sshd + ipv6_sshd instead?

Reply:
Fixed